### ISSUE TITLE
HyperLinkButton does not provide OnPointerOver brush/feedback in High Contrast Mode

### ISSUE THREAD

**Issue by niels9001** [microsoft/microsoft-ui-xaml#5016]
**Describe the bug**
The HyperLinkButton does not provide any feedback when hovering over it with the mouse. A HyperLink (part of a TextBlock) does. It should provide some feedback that it's an interactive element. This is an accessibility issue flagged here: https://github.com/microsoft/PowerToys/issues/10735

Here's a GIF that shows that the HyperLinkButtons used in Windows 10 Settings shows the correct behavior - where as the HyperLinkButton used in XAML Controls Gallery does not.
![HighContrast](https://user-images.githubusercontent.com/9866362/115115644-d5275400-9f95-11eb-8483-5e41b038bdf8.gif)

A HyperLink (within Text) does change color on mouse hover:
![HighContrastHyperLink](https://user-images.githubusercontent.com/9866362/115115839-e0c74a80-9f96-11eb-956b-16be8108eafb.gif)

**Steps to reproduce the bug**
1. Add HyperLinkButton to page
2. Turn on high contrast mode
3. Go over the HyperLinkButton with the mouse
4. See that nothing happens

**Expected behavior**
In highcontrastmode, the brush should change when hovering over the button so that it provides feedback that it's interactive and clickable. Now it seems disabled.

**Version Info**
2.5

**Comment by chingucoding** [comment]
I think we should the `SystemColorWindowTextColor` resource for hover and the `SystemColorHighlightColor` resource for pressed state (both are the same right now). @licanhua FYI

If it's clear what colors we want to use, I would love to fix this (unless @niels9001 is fine with this of course!).

**Comment by niels9001** [comment]
That would be great @chingucoding 😁! Much appreciated!

**Comment by StephenLPeters** [comment]
Seem right to me, @chigy thoughts?

**Comment by niels9001** [comment]
@chingucoding any plans by any chance to still pick this up :)?

**Comment by chingucoding** [comment]
Still awaiting for feedback from @chigy , @niels9001 :) @StephenLPeters Or is this good to go?

**Comment by chigy** [comment]
@chingucoding , this appears to have been already addressed in the series of changes we made recently.

**Comment by niels9001** [comment]
@chigy Not in the latest pre-release package, correct?

**Comment by niels9001** [comment]
![HyperlinkButton](https://user-images.githubusercontent.com/9866362/126644667-de6729bd-0e6b-493d-bf7d-7385faab004a.gif)

This is the latest pre-release package in a UWP app

**Comment by chigy** [comment]
> @chigy Not in the latest pre-release package, correct?

Correct. I checked with the latest XCG that is in store which uses the latest release bits.

**Comment by niels9001** [comment]
@chigy any updates on this?

**Comment by niels9001** [comment]
@chigy I finally figured out why this wasn't working for PowerToys. [HighContrastMode ](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.uielement.highcontrastadjustment?view=winrt-19041) should be set to None to make this happen.
